docs/source/plotting/index.rst

```diff
@@ -410,24 +410,25 @@ Custom Colors
 Custom Color Cycle (Line / 1D plots)
 ####################################
 
-The Default Color Cycle doesn't have to be used. Here is an example where a Custom Color Cycle is chosen. Make sure to fill the list `custom_colors` with the HTML codes for the desired colours. 
-These can be found `online <https://www.rapidtables.com/web/color/html-color-codes.html>`_ or using Mantid Workbench: On Workbench, plot a Spectrum and navigate to Figure Options(Gear Icon) > Curves > Line > Color (RGBA). Click on the nearby colorbox and use the 'Select Color' pop-up to chose your colors and their associated HTML codes. `Further Details <https://matplotlib.org/3.1.1/gallery/color/color_cycle_default.html>`_
+The Default Color Cycle doesn't have to be used. Here is an example where a Custom Color Cycle is chosen. Make sure to fill the list `custom_colors` with either the HTML codes or recognised names for the desired colours. 
```

Comment:
> HTML codes

I still don't think this is clear enough as HTML can support a range of formats for colour. If the colour code has to be in the form `#b3457f` then you should specify "HTML _hex_ code".

Reply:
Latest commit adds in hex
AND removes the ColorCycle Image as you are correct I'm now Plotting this.
The ColorMapCustomPlot.PNG is from MantidPlot, not using Matplolib so needs to remain as an image!